src/pages/signup-affiliates/components/_account-terms.tsx

```diff
@@ -1,14 +1,51 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
 import AgreementLabel from 'components/custom/_agreement-label'
 import { localize } from 'components/localization'
 import { Header } from 'components/elements'
 
-const AccountTerms = () => {
-    const [non_pep_declaration, setNonPepDeclaration] = useState(false)
-    const [tnc_accepted, setTncAccepted] = useState(false)
-    const [is_eu_checked, setEuChecked] = useState(false)
-    const [is_partner_checked, setPartnerChecked] = useState(false)
+type ConsentProps = {
+    non_pep_declaration: boolean
+    tnc_accepted: boolean
+    is_brokers_checked: boolean
+    is_eu_checked: boolean
+    is_partner_checked: boolean
+}
+
+type AccountTermsProps = {
+    updatedData: (e) => void
```

Comment:
```sugesstion
updateData: (configs: ConsentProps) : void
```
we can change `ConsentProps` to `TAffiliateTerms`, just sugession though :pray:

Reply:
okay